os/linux/iahwc.h

```diff
@@ -126,6 +127,13 @@ enum iahwc_layer_transform {
   IAHWC_TRANSFORM_FLIP_V_ROT_90
 };
 
+enum iahwc_power_modes {
+  IAHWC_POWER_MODE_ON,
```

Comment:
Instead of adding new ones, can we re-use these https://github.com/intel/IA-Hardware-Composer/blob/master/public/hwcdefs.h#L98 directly

Reply:
okay. Including that file directly into iahwc.h would require __cplusplus ifdef blocks. Should I add them there or any other better option?

Reply:
Or shall I use DRM_MODE_DPMS_* as the intermediary and then convert them to k{On,Off,...}?

Reply:
@harishkrupo Adding _cplusplus guards is fine